server/models/Notification.js

```diff
@@ -0,0 +1,38 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema(
+  {
+    user: {
```

Comment:
```suggestion
        receiver: {
           type: mongoose.Schema.Types.ObjectId,
           required: true,
           ref: "User",
        },
         sender: {
          type: mongoose.Schema.Types.ObjectId,
          required: true,
          ref: "User",
       },
```

Sorry I may have overlooked this the first time but we'll need some sort of reference to keep track of the creator and receiver for notifications. The example above is one possible way we can accomplish it :)